.github/workflows/pr-verification.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: PR verification
+on:
+  pull_request:
```

Comment:
I think we should include more [activity types](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request). For example, `edited`

Reply:
Added!